src/LmdbCache.ts

```diff
@@ -158,14 +202,8 @@ export class LmdbCache<V = any> implements SortKeyCache<V> {
     throw new Error('Not implemented yet');
   }
 
-  async keys(): Promise<string[]> {
-    const keys = this.db.getKeys();
-    const contracts = new Set<string>();
-    keys.forEach((k) => {
-      contracts.add(k.split('|')[0]);
-    });
-
-    return Array.from(contracts);
+  async keys(sortKey?: string, options?: SortKeyCacheRangeOptions): Promise<string[]> {
+    return Array.from((await this.kvMap(sortKey, options)).keys());
```

Comment:
The type of `sortKey` in `kvMap` is `sortKey: string`, but we are providing the type `sortKey?: string` to that argument in this call. Is this safe?

Reply:
I think I missed something here, I'll look into it :/